src/FormBuilder.php

```diff
@@ -32,6 +32,16 @@ public function add(Closure $step, ?string $name = null, bool $ignoreWhenReverti
         return $this;
     }
 
+    /**
+     * Add a new conditional step.
+     */
+    public function addIf(Closure $condition, Closure $step, ?string $name = null, bool $ignoreWhenReverting = false): self
```

Comment:
```suggestion
    public function addIf(bool|Closure $condition, Closure $step, ?string $name = null, bool $ignoreWhenReverting = false): self
```

Reply:
What is the purpose of having a step with a false condition? Why would you add such a step?